src/anndata/_core/anndata.py

```diff
@@ -675,6 +670,52 @@ def X(self, value: np.ndarray | sparse.spmatrix | SpArray | None):
     def X(self):
         self.X = None
 
+    __obsm__alignedmapping_property = AlignedMappingProperty("obsm", AxisArrays, 0)
+
+    @property
+    def obsm(self) -> AxisArrays | AxisArraysView:
+        """\
+        Multi-dimensional annotation of observations
+        (mutable structured :class:`~numpy.ndarray`).
+
+        Stores for each key a two or higher-dimensional :class:`~numpy.ndarray`
+        of length `n_obs`.
+        Is sliced with `data` and `obs` but behaves otherwise like a :term:`mapping`.
+        """
+        return self.__obsm__alignedmapping_property
+
+    @obsm.setter
+    def obsm(self, val):
+        self.__obsm__alignedmapping_property = val
+
+    @obsm.deleter
+    def obsm(self):
+        del self.__obsm__alignedmapping_property
```

Comment:
You defined an accessor, why not use it directly?

```suggestion
    obsm = AlignedMappingProperty("obsm", AxisArrays, 0)
    """\
    Multi-dimensional annotation of observations
    (mutable structured :class:`~numpy.ndarray`).
    Stores for each key a two or higher-dimensional :class:`~numpy.ndarray`
    of length `n_obs`.
    Is sliced with `data` and `obs` but behaves otherwise like a :term:`mapping`.
    """
```

Reply:
done, needed a small correction for Sphinx, that’s all.

Reply:
This was the reason for the boilerplate.  I couldn't get `sphinx` to work with this without some amount of pain, and then the type hints also didn't work.

Reply:
Now typing works, only the types in e.g. `obsm`’s documentation doesn’t work. But I think that’s fine for now.

Reply:
yup, we sure can